sentences/de/_common.yaml

```
@@ -280,8 +280,8 @@ expansion_rules:
   schalten: "(schalt[e|en|et])"
   machen: "(mach[e|en|t])"
   setzen: "(setz[e|en]|stell[e|en]|einstellen|änder[e|n]|veränder[e|n])"
-  abdeckung: "<artikel> (Rollo[s]|Abdeckung[en]|Rolll(a|ä)den|Jalousie[n]|Raffstore[s]|Markise[n]|Vorh(a|ä)ng[e])"
-  alle_abdeckungen: "<alle> (Rollos|Abdeckungen|Rolll(a|ä)den|Jalousien|Raffstores|Markisen|Vorhänge)"
+  abdeckung: "<artikel> (Rollo[s]|Abdeckung[en]|Rolll(a|ä)den|Jalousie[n]|Raffstore[s]|Markise[n]|Vorh(a|ä)ng[e]|Beschattung[en])"
+  alle_abdeckungen: "<alle> (Rollos|Abdeckungen|Rolll(a|ä)den|Jalousien|Raffstores|Markisen|Vorhänge|Beschattung[en])"
```

```suggestion
  alle_abdeckungen: "<alle> (Rollos|Abdeckungen|Rolll(a|ä)den|Jalousien|Raffstores|Markisen|Vorhänge|Beschattungen)"
```
we don't need the singular in here